### ISSUE TITLE
Custom SSL vhost setting option should be active by default

### ISSUE THREAD

**Issue by tnebe** [Froxlor/Froxlor#739]
# Summary
With froxlor 0.10.2 an option has been introduced to set different custom vhost settings per domain for SSL and non-SSL configurations.
Before that only one custom configuration was applied to both vhost container.

This causes an issue when performing the upgrade - existing SSL domains lose the previous set custom vhost configuration.

Therefore the option to share the config from non-SSL host should be checked by default.

# System information
* Froxlor version: 0.10.2
* Web server: apache2

**Comment by d00p** [comment]
When updating from 0.10.1 to 0.10.2 (when this was introduced) for UPDATERS the `include_specialsettings` option is `enabled` for all ssl IP's which have content in `specialsettings` - so I don't understand why this is an issue. For new users and new IP/Ports this can be explicitly set. That's why this is two fields and _optional_ to use one for both.

**Comment by tnebe** [comment]
The option was definetely not set to `enabled` for all of my domains where I had defined `specialsettings` inside. I updated from 0.9.40 to 0.10.2.

I feel it would make more sense to have this option activated by default because I think this is mthe more expected behaviour - at least from my use cases. But this is your decision.

Ticket can be closed if that issue only appeared for my setup.

many thanks for all your efforts 👍
